packager/live_packager.cc

```diff
@@ -247,7 +249,7 @@ Status LivePackager::PackageInit(const Segment& init_segment,
 
   shaka::PackagingParams packaging_params;
   packaging_params.chunking_params.segment_duration_in_seconds =
-      config_.segment_duration_sec;
+      SEGMENT_DURATION;
```

Comment:
Why do we need a default value at all ?

Reply:
Shaka requires a non-zero value for segment duration otherwise it throws an error.  For our use-case of packaging segments individually, this value has no effect.

Reply:
Done